auth/src/main/java/com/firebase/ui/auth/ui/account_link/WelcomeBackPasswordPrompt.java

```diff
@@ -83,8 +83,6 @@ protected void onCreate(Bundle savedInstanceState) {
         TextView bodyTextView = ((TextView) findViewById(R.id.welcome_back_password_body));
         bodyTextView.setText(spannableStringBuilder);
 
-        ((TextInputLayout) findViewById(R.id.password_layout)).setPasswordVisibilityToggleEnabled(false);
```

Comment:
See the support lib [release notes](https://developer.android.com/topic/libraries/support-library/revisions.html). In 25.0.1, they set the default to false.